docker/compose/kafka-schema-registry.yml

```diff
@@ -21,6 +21,22 @@ services:
       KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
       KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
 
+  init-kafka-container:
```

Comment:
@authorjapps  have added an init container that will create a multi partition topic to run the test against.

Ideally we should change the `num.partitions` setting in the kafka broker, but can't find a way to do that in the cp-kafka image we have used. Let me know your thoughts

Reply:
> @authorjapps have added an init container that will create a multi partition topic to run the test against.
> 
> Ideally we should change the `num.partitions` setting in the kafka broker, but can't find a way to do that in the cp-kafka image we have used. Let me know your thoughts


ok. Let's figure out this in the coming days.

Also,
if you have tested locally in your Kafka cluster that everything works as expected, then we should be fine for now. 
I will raise a TECH-DEBT to get a Integration test added to our CI suite to cover this `PARTITION->OFFSET NPE` scenario.

Let's get this merged anyways.